react/features/recording/functions.js

```diff
@@ -119,6 +120,65 @@ export function getSessionStatusToShow(state: Object, mode: string): ?string {
     return status;
 }
 
+/**
+ * Returns the recording button props.
+ *
+ * @param {Object} state - The redux state to search in.
+ *
+ * @returns {{
+ *    _disabled: boolean,
+ *    _tooltip: string,
+ *    visible: boolean
+ * }}
+ */
+export function getRecordButtonProps(state: Object): ?string {
+    let visible;
```

Comment:
in the original implementation of `AbstractRecordButton` the visible flag would only be processed if not passed on to the component by the embedding component (an ownProp). In the current approach seems that passing the visible flag as own prop is not considered.

Reply:
nevermind, seems there's no place where visible is passed on.